Subject: foo
From: olh@suse.de
Patch-mainline: never

drivers/md/raid6altivec1.c: In function `raid6_altivec1_gen_syndrome':
drivers/md/raid6altivec1.c:99: warning: implicit declaration of function `enable_kernel_altivec'
drivers/md/raid6altivec1.c: In function `raid6_have_altivec':
drivers/md/raid6altivec1.c:111: error: request for member `cpu_features' in something not a structure or union

--- ../linux-2.6.10-bk13/drivers/md/Kconfig	2004-12-24 22:33:59.000000000 +0100
+++ ./drivers/md/Kconfig	2005-01-10 18:17:50.885447957 +0100
@@ -127,7 +127,7 @@ config MD_RAID5
 
 config MD_RAID6
 	tristate "RAID-6 mode (EXPERIMENTAL)"
-	depends on BLK_DEV_MD && EXPERIMENTAL
+	depends on BLK_DEV_MD && EXPERIMENTAL && !PPC32
 	---help---
 	  WARNING: RAID-6 is currently highly experimental.  If you
 	  use it, there is no guarantee whatsoever that it won't
